### PR TITLE
Feature/lab04 update

### PR DESCRIPTION
**Pull request by vabarbosa**
Related: https://github.com/Qiskit/platypus/pull/232
Closes https://github.com/Qiskit/platypus/pull/295

- add a new Lab 04 and re-order/rename subsequent labs

preview: https://platypus-pr-295.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/course/ch-labs/lab-4-bell-ghz-circuit

### CODE REVIEW

**Comment by vabarbosa** [comment]
@frankharkins this should be all set, you can check out the preview link and give your final approval for merging (thanks):

https://platypus-pr-295.ecy0akwlcpw.us-south.codeengine.appdomain.cloud/course/ch-labs/lab-4-bell-ghz-circuit

**Review by techtolentino — COMMENTED** [review]
Hello, 
just wanted to call out one thing I noticed when looking at the preview branch — a small typo:

Screen Shot 2022-01-25 at 10 07 02 AM

**Comment by vabarbosa** [comment]
> Hello, just wanted to call out one thing I noticed when looking at the preview branch — a small typo:
> 

good catch! thanks @techtolentino

File: notebooks/ch-labs/Lab04_Bell_GHZ_Circuit.ipynb [hunk 1]
```diff
@@ -0,0 +1,896 @@
...
+    "#from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister\n",
+    "#from qiskit import IBMQ, Aer, transpile, assemble\n",
+    "#from qiskit.visualization import plot_histogram, plot_bloch_multivector, plot_state_qsphere\n",
+    "\n",
+    "# Importing standard Qiskit libraries\n",
+    "from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit\n",
+    "from qiskit import Aer, assemble\n",
+    "from qiskit.visualization import plot_histogram, plot_bloch_multivector, plot_state_qsphere, plot_state_city, plot_state_paulivec, plot_state_hinton\n",
+    "\n",
+    "# Ignote warnings\n",
```
Comment (vabarbosa) [inline notebooks/ch-labs/Lab04_Bell_GHZ_Circuit.ipynb]:
```suggestion
    "# Ignore warnings\n",
```

Comment (vabarbosa) [inline notebooks/ch-labs/Lab04_Bell_GHZ_Circuit.ipynb]:
```suggestion
    "# Ignore warnings\n",
```

**Comment by gunchamalik** [comment]
@techtolentino thanks for catching the type.

@vabarbosa May I also request you to remove the unwanted commented imports. they seem to be have been left mistakenly.
image

**Comment by vabarbosa** [comment]
> @vabarbosa May I also request you to remove the unwanted commented imports. they seem to be have been left mistakenly. 

@gunchamalik thanks. done.

@frankharkins if this is OK to merge? thanks

**Review by frankharkins — APPROVED** [review]

**Pull request merged**